pxtlib/docsrender.ts

```diff
@@ -418,13 +418,27 @@ namespace pxt.docs {
 
     export function setupRenderer(renderer: marked.Renderer) {
         renderer.image = function (href: string, title: string, text: string) {
-            let out = '<img class="ui image" src="' + href + '" alt="' + text + '"';
-            if (title) {
-                out += ' title="' + title + '"';
+
+            if (href.startsWith("https://www.youtube.com/embed/")){
+
+                // <iframe width="1358" height="773" src="https://www.youtube.com/embed/0Gl2QnHNpkA?list=RDMM8H1iMq0g31g" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
nit: is this comment still useful?

Reply:
good point, thanks for comment, will fix!